src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.html {
+  font-family: "Avenir", sans-serif;
+
+}
+
+.big,
+.small {
+  width: 100%;
+  position: relative;
+  background: #fff;
+}
+
+.big__img {
+  position: absolute;
+  top: 45px;
+  left: 26px;
+  width: 19px;
+}
+
+.small__img {
+  position: absolute;
+  top: 16px;
+  left: 13px;
+  width: 11px;
+}
+
+.small__search,
+.big__search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.small__search::placeholder,
+.big__search::placeholder {
+  color: #3d4e61;
+}
+
+.big__search::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  font-family: inherit;
+}
+
+.small__search::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: inherit;
+}
+
+.big__search {
+  margin: 20px 0;
```

Comment:
looks like the reason of failed tests is margin for input.
remove here and add margin for form